src/Platforms/SqlitePlatform.php

```diff
@@ -920,7 +920,8 @@ public function getAlterTableSQL(TableDiff $diff)
                 continue;
             }
 
-            $columns = $this->replaceColumn($diff->name, $columns, $oldColumnName, $column);
+            $oldColumnName = strtolower($oldColumnName);
```

Comment:
Note, the `replaceColumn()` method does convert the names to the lower case because this is how `$columns` are indexed. With this patch, it seems no longer necessary.

Reply:
I have removed the conversion to lower case in the `replaceColumn()` method